subprojects/packagefiles/flatbuffers/meson.build

```diff
@@ -1,25 +1,15 @@
-project(
-    'flatbuffers',
-    'cpp',
-    version : '2.0.6',
-    license : 'Apache-2.0'
-)
+project('flatbuffers', 'cpp', version: '2.0.8', license: 'Apache-2.0', meson_version: '>=0.49.0')
 
 cpp = meson.get_compiler('cpp')
 
 cpp_args = []
 
 if host_machine.system() == 'windows'
-    cpp_args += '-D_CRT_SECURE_NO_WARNINGS'
+  cpp_args += '-D_CRT_SECURE_NO_WARNINGS'
 endif
 
 if cpp.get_argument_syntax() == 'gcc'
-    cpp_args += [
-        '-Wall',
-        '-pedantic',
-        '-Werror',
-        '-Wextra'
-    ]
+  cpp_args += ['-Wall', '-pedantic', '-Werror', '-Wextra']
```

Comment:
These really should not be here. Meson has builtins for all of these and libraries should not enable `-Werror` unconditionally because it breaks downstream users that have e.g. newer compilers that have more checks that what the developers had available.

Reply:
Yeah probably. I didn’t bother removing them. Will do so.

Reply:
Fixed.